### ISSUE TITLE
Training loss is larger than validation loss (even test loss) due to the partition of dataset

### ISSUE THREAD

**Issue by PhoebeChen123** [cure-lab/SCINet#22]
Have you ever met such problem? And train loss could not converge to a value near 0 even if it has been training for many many epochs

**Comment by VEWOXIC** [comment]
May I know which dataset you trained on? 
Thanks for your attention.

**Comment by PhoebeChen123** [comment]
> May I know which dataset you trained on? Thanks for your attention.

ETTh1 dataset under univariate experiment

**Comment by ailingzengzzz** [comment]
Hi @PhoebeChen123,

Training loss is not close to 0 is a normal phenomenon, indicating that the current model can not fit the training data well. Instead, we pay more attention to whether the test set can achieve better results.
